### PR TITLE
Complex autograd doc fix

### PR DESCRIPTION
**Pull request by anjali411**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#46258 Complex autograd doc fix**

Differential Revision: [D24286512](https://our.internmc.facebook.com/intern/diff/D24286512)

### CODE REVIEW

File: docs/source/notes/autograd.rst [hunk 1]
```diff
@@ -222,7 +222,7 @@ The short version:
   the gradients are computed under the assumption that the function is a part of a larger real-valued
   loss function :math:`g(input)=L`. The gradient computed is :math:`\frac{\partial L}{\partial z^*}`
   (note the conjugation of z), which is precisely the direction of the step
-  you should take in gradient descent. Thus, all the existing optimizers work out of
+  you should take in gradient ascent. Thus, all the existing optimizers work out of
```
Comment (albanD) [inline docs/source/notes/autograd.rst:225]: While it is true, I don't think anyone is doing gradient ascent anymore :D 
I think that not specifying is better here: "which is precisely the direction expected by gradient-based optimization algorithms." What do you think?

Comment (anjali411) [reply docs/source/notes/autograd.rst:225]: yeah that sounds good!

File: docs/source/notes/autograd.rst [hunk 2]
```diff
@@ -221,8 +221,8 @@ The short version:
 - When you use PyTorch to differentiate any function :math:`f(z)` with complex domain and/or codomain,
   the gradients are computed under the assumption that the function is a part of a larger real-valued
   loss function :math:`g(input)=L`. The gradient computed is :math:`\frac{\partial L}{\partial z^*}`
-  (note the conjugation of z), which is precisely the direction of the step
-  you should take in gradient descent. Thus, all the existing optimizers work out of
+  (note the conjugation of z), the negative of which is precisely the direction of steepest descent
+  in Gradient Descent algorithm.. Thus, all the existing optimizers work out of
```
Comment (albanD) [inline docs/source/notes/autograd.rst:225]: nit: missing a "used"

**Review by albanD — APPROVED** [review]
lgtm

File: docs/source/notes/autograd.rst [hunk 3]
```diff
@@ -221,8 +221,8 @@ The short version:
 - When you use PyTorch to differentiate any function :math:`f(z)` with complex domain and/or codomain,
   the gradients are computed under the assumption that the function is a part of a larger real-valued
   loss function :math:`g(input)=L`. The gradient computed is :math:`\frac{\partial L}{\partial z^*}`
-  (note the conjugation of z), which is precisely the direction of the step
-  you should take in gradient descent. Thus, all the existing optimizers work out of
+  (note the conjugation of z), the negative of which is precisely the direction of steepest descent
+  used in Gradient Descent algorithm.. Thus, all the existing optimizers work out of
```
Comment (zou3519) [inline docs/source/notes/autograd.rst:225]: nit: two periods

**Pull request closed without merging**